contracts/0.4.24/nos/NodeOperatorsRegistry.sol

```diff
@@ -191,7 +199,7 @@ contract NodeOperatorsRegistry is INodeOperatorsRegistry, IsContract, AragonApp
     function updateUsedKeys(uint256[] _ids, uint64[] _usedSigningKeys) external onlyPool {
         require(_ids.length == _usedSigningKeys.length, "BAD_LENGTH");
         for (uint256 i = 0; i < _ids.length; ++i) {
-            require(_ids[i] < totalOperatorsCount, "NODE_OPERATOR_NOT_FOUND");
+            require(_ids[i] < getNodeOperatorsCount(), "NODE_OPERATOR_NOT_FOUND");
```

Comment:
I'd cache the result of `getNodeOperatorsCount()` here to avoid doing multiple `SLOAD`s per loop. Anyway, this function was removed in #226.

Reply:
Understood, if fun was removed, no need to fix that. I just will leave it as is.